news/8896.feature

```diff
@@ -0,0 +1 @@
+Move the batched download of lazily-fetched wheel files out of the ``Resolver.resolve()`` codepath, into ``PartialRequirementDownloadCompleter``. Consume this in the ``pip download``, ``pip wheel``, and ``pip install`` tasks.
```

Comment:
This fragment is user facing, so phrase it in terms of how it might affect the end user's experience. :)

Reply:
Not until we can implement parallel downloads, I think!! So `.trivial` seems quite appropriate.